test/test_conf_bash.sh

```diff
@@ -90,9 +90,9 @@ _test_androidpic_mv() {
     done
     for f in "${LIST_FILES_ALL[@]}"; do
         if [ ! -e "$f" ]; then
-            echo "[ERROR] $f is missing from the target folder."; ls -al $f; RESULT=1;
+            echo "[ERROR] $f is missing from the target folder."; ls -al $f;
         else
-            echo "[SUCCESS] $f is found at the target folder."; ls -al $f;
+            echo "[SUCCESS] $f is found at the target folder."; ls -al $f; RESULT=0;
```

Comment:
This change is BAD. This makes (almost) all tests pass, even if there was a failure during the loop.